tests/samplesheets.nf.test

```diff
@@ -35,6 +36,22 @@ nextflow_pipeline {
             ],
             failure: true,
             stderr: [1..2]
+        ],
+        [
+            name: "-profile test --input tests/csv/3.0/multiple_lane_ids.csv",
+            params: [
+                input: "${projectDir}/tests/csv/3.0/multiple_lane_ids.csv",
+            ],
+            failure: true,
+            stderr: [0..-1]
+        ],
+        [
```

Comment:
sorry for being annoying about this, but can we get one succcessful test as well? Or is that already covered by other tests?